package.json

```diff
@@ -25,7 +25,7 @@
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "start": "ember serve",
-    "test": "ember test && ember fastboot:test",
```

Comment:
I see in the fork this was removed as well.  Not sure of the context but I'm having trouble getting the tests to pass/even run with the addon fastboot tests locally and CI.

Reply:
scratch that.  It works locally.  Just not in CI